code/ship/shiphit.cpp

```diff
@@ -1356,7 +1356,7 @@ static void player_died_start(object *killer_objp)
 
 //#define	DEATHROLL_TIME						3000			//	generic deathroll is 3 seconds (3 * 1000 milliseconds) - Moved to ships.tbl
 #define	MIN_PLAYER_DEATHROLL_TIME		1000			// at least one second deathroll for a player
-#define	DEATHROLL_ROTVEL_CAP				6.3f			// maximum added deathroll rotvel in rad/sec (about 1 rev / sec)
+//#define	DEATHROLL_ROTVEL_CAP				6.3f			// maximum added deathroll rotvel in rad/sec (about 1 rev / sec) - moved to shiphit.h & converted to const float
```

Comment:
I'd prefer to remove the code instead of commenting it out.
Just add a note in your commit message that you moved DEATHROLL_ROTVEL_CAP etc.
People can look it up there and possibly restore it if required from git.

Reply:
will do